tasks.py

```diff
@@ -1,11 +1,13 @@
 import logging
 import platform
+from datetime import date, timedelta, datetime
 
 import urllib
 from invoke import run, task
 from elasticsearch import helpers
 
 import scrapi.harvesters  # noqa
+from dateutil import parser
```

Comment:
I think in the rest of the codebase we have ```from dateutil.parser import parse```. Sort of nitpicky at this point, so you don't need to change it if you don't want to.